FRUI/FRUI/ViewControllers/AuthStepViewController.swift

```diff
@@ -255,6 +263,22 @@ class AuthStepViewController: UIViewController {
             return
         }
         
+        var listItems: [Any] = []
+        for callback in self.authCallbacks {
+            if let idpCallback = callback as? SelectIdPCallback {
+                for provider in idpCallback.providers {
+                    if provider.provider != "localAuthentication" {
```

Comment:
I would suggest against comparing against hardcoded strings. If there is no better way to identify the provider then maybe we should have that as part of a Providers enum helper

Reply:
Same thing here. IdPValue.provider is a plain text input from the end-user, so enum may not be helpful. For this implementation, I have not incorporated the Callback constants that you've suggested for the previous feature testing. I am planning to apply that constant values later when I merge this repo into the stash repo.

Reply:
Same as above Callback constants is OK with me.